test/slowlyChangingColorsTest.java

```diff
@@ -35,17 +35,14 @@ private Color slowlyChangingColors(String color) {
 			if (c == '8') return new Color(55, 55, 55);
 			if (c == '9') return new Color(65, 65, 65);
 		}
-		if (color == "RAINBOW") {
-			if (c == '0') return new Color(254, 254, 0);
-			if (c == '1') return new Color(135, 254, 0);
-			if (c == '2') return new Color(0, 254, 0);
-			if (c == '3') return new Color(0, 254, 254);
-			if (c == '4') return new Color(0, 135, 254);
-			if (c == '5') return new Color(0, 0, 254);
-			if (c == '6') return new Color(135, 0, 205);
-			if (c == '7') return new Color(254, 0, 224);
-			if (c == '8') return new Color(254, 0, 135);
-			if (c == '9') return new Color(220, 200, 254);
+		if (color == "RANDOM") {
```

Comment:
[P2] Good job in making the title colorful, I like it. I don't see a difference in result between rainbow and random, but either way it's good. Also, yellow is used twice in the new code: rndColor[0] = (240, 240, 60) and rndColor[4] = (240, 230, 50). Perhaps change one to a different color?

Reply:
That's right, I implemented the feature by refactoring the color algorithm when ```color == RAINBOW```.
It was an effect that had not been applied before, so I applied it to the game after making some modifications.

As you suggested, the color has been changed.

Thank you for your feedback!